jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/LambdaHandlerIndexTest.kt

```diff
@@ -259,18 +257,4 @@ class LambdaHandlerIndexTest {
             assertThat(LambdaHandlerIndex.listHandlers(projectRule.project)).isEmpty()
         }
     }
-
-    @Test
-    fun testVersionChangesIfExtensionsChange() {
-        val initialVersion = LambdaHandlerIndex().version
-
-        val extensionPointName =
-            ExtensionPointName.create<RuntimeGroupExtensionPoint<LambdaHandlerResolver>>("aws.toolkit.lambda.handlerResolver")
-
-        PlatformTestUtil.unregisterAllExtensions(extensionPointName, projectRule.fixture.testRootDisposable)
-
-        val newVersion = LambdaHandlerIndex().version
-
-        assertThat(initialVersion).isNotEqualTo(newVersion)
-    }
```

Comment:
Why do we remove this test?

Reply:
The method that it used to mess with extensions have been removed, and the new methods arent in 2018.3

https://github.com/JetBrains/intellij-community/blob/master/platform/extensions/src/com/intellij/openapi/extensions/ExtensionPoint.java#L35

Reply:
i'll look again to see if I can replicate the behavior

Reply:
Was able to re-create the test manually:
```kotlin
    @Test
    @Suppress("DEPRECATION")
    fun testVersionChangesIfExtensionsChange() {
        val initialVersion = LambdaHandlerIndex().version

        val extensionPointName = ExtensionPointName.create<RuntimeGroupExtensionPoint<LambdaHandlerResolver>>("aws.toolkit.lambda.handlerResolver")
        val extensionPoint = extensionPointName.getPoint(null)
        val extensions = extensionPoint.extensions

        Disposer.register(projectRule.fixture.testRootDisposable, Disposable {
            extensions.forEach { extensionPoint.registerExtension(it) }
        })

        extensions.forEach { extensionPoint.unregisterExtension(it) }

        val newVersion = LambdaHandlerIndex().version

        assertThat(initialVersion).isNotEqualTo(newVersion)
    }
```